webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardList.vue

```diff
@@ -251,6 +251,15 @@ export default {
           this.initialized = true;
         });
 
+    },
+    waitForInitializing() {
+      window.setTimeout(() => {
+        if (this.initialized) {
+          this.searchPeople();
+        } else {
+          this.waitForInitializing();
+        }
+      }, 50);
```

Comment:
```suggestion
```

Add a watch on `initialized` instead :

```javascript
watch: {
  initialized() {
    if (this.initialized) {
          this.searchPeople();
    }
  }
},
```

Reply:
BTW, it would be more elegant IMO to abort first fetch and execute right a way the user query by using :
https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort

```javascript
  if (this.controller) {
    this.controller.abort();
  }
  this.controller = new AbortController();,
  ...
  // Pseudo algorithm, this will allow to have `this.signal` variable in `getUsers` method without adding signal a argument which will break the API signature
  this.$userService.getUsers.apply({singal: this.controller.signal}, this.keyword, this.offset...)
  .finally(() => {
...
    this.controller = null;
  });
....
```